.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Thanks for opening a Pull Request (PR), here's a few guidelines as to what we need in your PR before we review it.
```

Comment:
typo: here are a few guidelines...

Reply:
ah, I knew it! But for some reasons, I convinced myself that this was written by a native speaker and so that it was correct..